mcsema/Arch/ABI.cpp

```diff
@@ -374,13 +379,14 @@ static const char *FloatReturnValVar(llvm::CallingConv::ID cc,
   return nullptr;
 }
 
-static const char *ReturnValVar(llvm::CallingConv::ID cc, llvm::Type *type) {
+static const char *ReturnValVar(llvm::CallingConv::ID cc, llvm::Type *type, size_t index = 0) {
```

Comment:
Remove spaces around `=` on `index = 0` for default argument value specifications.

Reply:
Also break line before last `,` because it's longer than 80 cols.